### DISCUSSION TITLE
Is genotype the same as "full genotype" in LabTracks?

### DISCUSSION THREAD

**Discussion by galenlynch** [AllenNeuralDynamics/aind-data-schema#645] [Q&A] [answered]
There are a number of things that look like genotypes in LabTracks (e.g. pedigree). I am assuming that "genotype" in the schema is the same as "full genotype" in LabTracks (and reflects genetic testing), but just wanted to check.

**Comment by saskiad** [comment, marked as answer]
yes, it is "full genotype"
